modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc

```diff
@@ -1,68 +1,34 @@
+:raymii-ssl-url: https://raymii.org/s/tutorials/Strong_SSL_Security_On_Apache2.html
+:certbot-url: https://certbot.eff.org
+:certbot-docs-url: https://eff-certbot.readthedocs.io/en/stable/
+:ssl-labs-url: https://www.ssllabs.com/ssltest/
+:crontab-guru-url: https://crontab.guru
+:snapcraft-url: https://snapcraft.io/docs/getting-started
 
 == Introduction
 
-This page covers how to configure your web server to use
-https://letsencrypt.org[Let’s Encrypt] as the certificate authority for
-your ownCloud server. Note that Let’s Encrypt is _not officially
-supported_, and this page is _community-maintained_.
-Thank you, contributors!
-
-* For ease of handling, SSL-specific directives have been moved into a
-separately included file. This can help for first-time certificate
-issuance as well as for reusing configurations.
-* The examples shown are based on Ubuntu 18.04.
-* Read the https://certbot.eff.org/docs/using.html[Certbot user guide]
-for details of the commands.
-* Let’s Encrypt CA issues short-lived certificates valid for 90 days.
-Make sure you renew the certificates at least once in this period,
-because expired certificates need reissuing. A certificate is due for
-renewal earliest 30 days before expiring. Certbot can be forced to renew
-via options at any time as long the certificate is valid.
-
-TIP: Raymii.org provides https://raymii.org/s/tutorials/Strong_SSL_Security_On_Apache2.html[an excellent introduction to strong SSL security measures with Apache], if you would like to know more.
+This page covers how to configure your web server to use {letsencrypt-url}[Let’s Encrypt] as the certificate authority for your ownCloud server. Note that Let’s Encrypt is _not officially supported_, and this page is _community-maintained_.
 
-== Requirements & Dependencies
-
-You require a domain name with a valid
-https://support.dnsimple.com/articles/a-record/[A-Record] pointing back
-to your servers IP address. In case your server is behind a firewall, take
-the necessary measures to ensure that your server is accessible, worldwide,
-from the internet, by adding the required firewall and port forward rules.
-
-== Install Let’s Encrypt’s Certbot Client
-
-The https://certbot.eff.org[Certbot] client can be installed in following ways:
-
-1. Installation via xref:via-apt[APT install]
-2. With the Ubuntu xref:via-ppa[PPA repository].
+* For ease of handling, SSL-specific directives have been moved into a separately included file. This can help for first-time certificate issuance as well as for reusing configurations.
+* Read the {certbot-docs-url}[Certbot user guide] for details of the commands.
+* Let’s Encrypt CA issues short-lived certificates valid for 90 days. Make sure you renew the certificates at least once in this period, because expired certificates need reissuing. A certificate is due for renewal earliest 30 days before expiring. Certbot can be forced to renew via options at any time as long the certificate is valid.
```

Comment:
```suggestion
* Let’s Encrypt CA issues short-lived certificates valid for 90 days. Make sure you renew the certificates at least once in this period, because expired certificates need reissuing. A certificate is due for renewal at the earliest 30 days before expiring. Certbot can be forced to renew via options at any time as long the certificate is valid.
```